tests/defined_resource_types_spec.rb

```diff
@@ -71,7 +110,12 @@
   end
 end
 describe "Task 6:", host: :pastureappsmall do
-  it 'Trigger a Puppet run on pasture-app-small.puppet.vm to enforce your changes' do
+  it 'has a working solution', :solution do
+    command("puppet job run --nodes pasture-app-small.puppet.vm")
```

Comment:
@kjhenner The `puppet job` command should run on the LVM proper, not inside the Docker container. Is there a way to override the target host for a single `it` statement inside of the `describe` statement? If not, I assume that means another `describe` statement would be required, but would it be named `Task 6a` or something like that?